.github/workflows/regression-tests.yml

```diff
@@ -36,6 +36,7 @@ jobs:
           ls -l ..
 
       - name: Run regression tests action
+        timeout-minutes: 45
```

Comment:
Maybe we should increase this a little bit more? What do you say?

Reply:
I think 45 is enough for for now